src/Factories/FunFunFactory.php

```diff
@@ -1130,6 +1140,22 @@ public function newDonationConfirmationPresenter(): DonationConfirmationHtmlPres
 		);
 	}
 
+	public function newDonorUpdatePresenter(): DonorUpdateHtmlPresenter {
+		return new DonorUpdateHtmlPresenter(
+			$this->getChoiceFactory()->getConfirmationPageTemplate(
+				$this->getSkinTwig(),
+				array_merge(
+					$this->getDefaultTwigVariables(),
+					[
+						'piwikGoals' => [ 3 ],
```

Comment:
What does `3` mean and is it the correct value for this page?

Reply:
Goal 3 is the custom goal configured in Matomo for "completed", I assume that we do not want to re-send statistics for that goal so I will remove it here.